assets/src/edit-story/components/helpCenter/constants.js

```diff
@@ -105,6 +106,21 @@ export const TIPS = {
       ),
     ],
   },
+  [KEYS.EMBED_WEB_STORIES]: {
+    title: __('Embed Web Stories', 'web-stories'),
+    /* `figureSrcImg` and `figureAlt` are temporary until
+    we get an animation. Once we have the animation then these
+    attributes should be removed. */
```

Comment:
```suggestion
    /* TODO #7212 `figureSrcImg` and `figureAlt` are temporary until
    we get an animation. Once we have the animation then these
    attributes should be removed. */
```

Created a follow up ticket to swap this out for the animation: https://github.com/google/web-stories-wp/issues/7212

Reply:
oh thank you!